langgraph/src/pregel/io.ts

```diff
@@ -9,19 +57,91 @@ export function* mapInput(
   chunk?: any
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
 ): Generator<[string, any]> {
-  if (typeof inputChannels === "string") {
-    yield [inputChannels, chunk];
+  if (chunk) {
```

Comment:
This check was added in the Python implementation: https://github.com/andrewnguonly/langgraph/blob/main/langgraph/pregel/io.py#L52